test/test_models.rb

```diff
@@ -3,6 +3,9 @@
 
   def persisted?; false; end
   def to_model; self; end
+  def to_param; "1"; end
```

Comment:
Use empty lines between defs.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.